install.sh

```diff
@@ -7,8 +7,14 @@ ORACLE_RPM="$(basename $ORACLE_FILE .zip)"
 
 cd "$(dirname "$(readlink -f "$0")")"
 
-sudo apt-get -qq update
-sudo apt-get --no-install-recommends -qq install bc libaio1 rpm unzip
+deps="bc libaio1 rpm unzip"
+if dpkg -s $deps >/dev/null 2>/dev/null; then
+  echo "Oracle XE dependencies are already installed: $deps"
+else
+  echo "Installing Oracle XE dependencies: $deps"
+  sudo apt-get -qq update
+  sudo apt-get --no-install-recommends -qq install $deps
```

Comment:
I believe we should update the setup documentation to promote use of `addons: apt:`, as it is quicker, and dont mind if you want to completely remove this feature from `install.sh`.

However I think it is worthwhile to keep this feature in `install.sh`,  so that it is more resilient if the user hates messing with `.travis.yml`.

Currently `travis-oracle` can be used by just adding a few lines in one of the `script:` blocks, modifying the maybe `sudo:` setting (not necessary if they use `sudo` in the build script`), and stuffing all the env vars into the Settings panel that Travis provides in the web interface, or supplying them in another script like https://github.com/lebretr/test-travis-oracle/blob/master/.travis/sequelize/init.sh.